README.md

```diff
@@ -151,6 +151,33 @@ This will create a clean environment where is uses the [pip-tools](https://githu
 $ make test
 ```
 
+## Production
+
+In order to get the container ready for production use we need to first build it:
+
+```bash
+$ docker build -t seismic-risc:latest .
+```
+
+Use the `prod.env.dist` template file and create a `prod.env` file with the correct environment variables and run like so:
+
+```bash
+$ docker run --env-file prod.env code4romania/seismic-risc "./start.sh"
+```
+
+Or, you can provide all the environment variables at runtime:
+
+```bash
+$ docker run -e DJANGO_CONFIGURATION=Prod -e DJANGO_SECRET_KEY= -e DATABASE_URL=postgres://USER:PASSWORD@HOST:PORT/NAME -e GUNICORN_PORT=5000 -e GUNICORN_WORKERS=2 code4romania/seismic-risc "./start.sh"
+```
+
+After testing the container runs properly, tag and upload the image to Docker hub:
+
+```bash
+$ docker tag seismic-risc:latest code4romania/seismic-risc:latest
+$ docker push code4romania/seismic-risc:latest
```

Comment:
Not sure if this is the right way to push an image to https://hub.docker.com/r/code4romania/seismic-risc

Please take a look and let me know @RaduCStefanescu

Reply:
Just tested these commands on https://hub.docker.com/r/aramboi/seismic-risc and worked fine. :)

Reply:
@aramboi thank you for checking
I will mark the conversation as resolved.